charts/vector/ci/aggregator-all-values.yaml

```diff
@@ -41,6 +41,16 @@ affinity:
           - e2e-az1
           - e2e-az2
 
+topologySpreadConstraints:
+  - labelSelector:
+      matchLabels:
+        app.kubernetes.io/name: vector
+        app.kubernetes.io/instance: vector
+        app.kubernetes.io/component: Aggregator
```

Comment:
The use-case where the `topologySpreadConstraints.labelSelector` is the same as the label selectors used in other related API objects selecting the workload Pods is very common. It would be nice to be able to just say `{{ include "vector.selectorLabels" . }}` in the user-provided `values.yaml`.

I've tried tinkering around with the Helm [tpl function](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-tpl-function), but the results were not very ergonomic.